org-code-javabuilder/lib/src/main/java/org/code/javabuilder/LambdaLogHandler.java

```diff
@@ -0,0 +1,26 @@
+package org.code.javabuilder;
+
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import org.code.protocol.JavabuilderLogger;
+import org.json.JSONObject;
+
+public class LambdaLogHandler implements JavabuilderLogger {
+  private final String sessionId;
+  private final LambdaLogger logger;
+
+  public LambdaLogHandler(LambdaLogger logger, String sessionId) {
+    this.logger = logger;
+    this.sessionId = sessionId;
+  }
+
+  @Override
+  public void logInfo(JSONObject eventData) {
+    this.logger.log("INFO: " + eventData.toString());
+  }
```

Comment:
Do we need the session ID in `logInfo` too?

Reply:
per Suresh we will only need it for debugging student issues, so for info we can go without session id